packages/components/src/radio-group/README.md

```diff
@@ -0,0 +1,87 @@
+# RadioGroup
+
+Use a RadioGroup component when you want users to select one option from a small set of options.
+
+![RadioGroup component](https://wordpress.org/gutenberg/files/2018/12/s_96EC471FE9C9D91A996770229947AAB54A03351BDE98F444FD3C1BF0CED365EA_1541792995815_ButtonGroup.png)
+
+## Table of contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+3. [Related components](#related-components)
+
+## Design guidelines
+
+### Usage
+
+#### Selected action
+
+Only one option in a radio group can be selected and active at a time. Selecting one option deselects any other.
+
+### Best practices
+
+Radio groups should:
+
+- **Be clearly and accurately labeled.**
+- **Clearly communicate that clicking or tapping will trigger an action.**
+- **Use established colors appropriately.** For example, only use red buttons for actions that are difficult or impossible to undo.
+- **Have consistent locations in the interface.**
+- **Have a default option already selected.**
+
+### States
+
+#### Active and available radio groups
+
+A radio group’s state makes it clear which option is active. Hover and focus states express the available selection options for buttons in a button group.
+
+#### Disabled radio groups
+
+Radio groups that cannot be selected can either be given a disabled state, or be hidden.
```

Comment:
I understand this as if I pass a `disabled` prop to `RadioGroup` it'll make the `Radio` buttons disabled as well, which I don't think is true in this case. Could you clarify this?

Reply:
Yep, I missed that. It's added now in 538538f along with a storybook example.